lightwood/mixer/regression.py

```diff
@@ -12,11 +12,21 @@
 
 
 class Regression(BaseMixer):
+    """
+    A mixer that runs a simple linear regression using the (Encoded) features to predict the target.
+    Supports all types because they are all encoded numerically.
+    """ # noqa
     model: LinearRegression
     label_map: dict
     supports_proba: bool
 
     def __init__(self, stop_after: int, target_encoder: BaseEncoder, dtype_dict: dict, target: str):
+        """
+        :param stop_after: Maximum amount of time it should train for, currently ignored
```

Comment:
in seconds? minutes? add the time interval.

Reply:
Oh, good point. This is an issue in like half a dozen places (we don't specify) the unit. So I went ahead and did a more generic fix where `stop_after` and `time_aim` are now typed as `seconds` so that it's obvious what they are all the time.